test/widgets/settings/settings_page_test.dart

```diff
@@ -0,0 +1,111 @@
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:mockito/mockito.dart';
+import 'package:motis_mitfahr_app/account/models/profile.dart';
+import 'package:motis_mitfahr_app/account/pages/about_page.dart';
+import 'package:motis_mitfahr_app/account/pages/account_page.dart';
+import 'package:motis_mitfahr_app/account/pages/help_page.dart';
+import 'package:motis_mitfahr_app/util/locale_manager.dart';
+import 'package:motis_mitfahr_app/util/profiles/profile_widget.dart';
+import 'package:motis_mitfahr_app/util/supabase.dart';
+import 'package:motis_mitfahr_app/util/theme_manager.dart';
+
+import '../../util/factories/profile_factory.dart';
+import '../../util/mocks/mock_server.dart';
+import '../../util/mocks/navigator_observer.mocks.dart';
+import '../../util/mocks/request_processor.mocks.dart';
+import '../../util/pump_material.dart';
+
+void main() {
+  final MockRequestProcessor processor = MockRequestProcessor();
```

Comment:
is the processor needed? I don't see requests

Reply:
no thanks, force of habit xD